portfolio/src/main/webapp/index.html

```diff
@@ -54,6 +55,10 @@ <h2>Hello, my name is Christian Liguori.</h2>
             <!-- TODO: Separate JS and HTML concerns. -->
             <button onclick="addRandomQuote()">Get a Quote!</button>
             <div id="quote-container"></div>
+            <br>
```

Comment:
Nit: I believe different browsers can display <br> differently so having multiple to convey space isn't the optimal solution & in many cases not maintainable, instead you can try to add padding or margins on your element to get the correct spacing. This is a nit though (so feel free to ignore as line breaks do work!) 

https://www.w3schools.com/css/css_padding.asp

Reply:
Okay, I did not know that. I looked it up and found this article ( https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br ) that said the intended use of the break element is to create a line break in a block of text. I never knew that. Thanks for pointing this out. I'll make the changes.